playback-service/src/main/java/com/playback/PlaybackStore.java

```diff
@@ -0,0 +1,57 @@
+package com.playback;
+
+import com.playback.models.*;
+import com.playback.services.ContentSecurityPolicyService;
+import com.playback.services.CustomerInfoService;
+import com.playback.services.SteeringService;
+import com.playback.services.VideoMetadataService;
+import com.playback.standards.VideoResolution;
+
+import java.util.Collection;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+public class PlaybackStore {
```

Comment:
Let's assume this class is instantiated once per container, what are some of the concerns?

Reply:
The `PlaybackStore` instance would be shared between concurrent requests. That would mean that `VideoMetadataService`, `CustomerInfoService`, `ContentSecurityPolicyService`, and `SteeringService` would need to be able to handle concurrent requests safely.